### PR TITLE
EN: Translations/fixes/grammar etc.

### PR DESCRIPTION
**Pull request by Aida-Enna**
Temporary PR, do not merge

### CODE REVIEW

**Comment by Aida-Enna** [comment]
Ready for merge/checking/etc

**Comment by alama** [comment]
:thumbsup:

File: Quests/Buster Quests/co_052103_01.csv [hunk 1]
```diff
@@ -4,6 +4,6 @@ linewindow#0,"""Mana collection completed!"""
 11_voice_navi09_war_044#0,"""We've collected enough Mana! With this, you should be able to summon the Buster Pile!"""
 11_voice_navi09_war_045#0,"""<%me>!<br>It seems the enemy's assault has begun! Let's concentrate on protecting the Buster Tower!"""
 text00#0,"""About the Defense Phase"""
-text01#0,"""<c 00ffff>Buster Piles<c> can destroy the devil barriers<br>created from the Devil Castle.<br><br>They are automatically summoned during the<br>Buster Phase and head towards the Devil Castle.<br><br><c ff4f3f>When all of the Buster Piles that are summoned<br>hit and destroy the Devil Barriers, or if they're destroyed,<br>the Buster Phase ends.<c>"""
+text01#0,"""<c 00ffff>Buster Piles<c> can destroy the Devil Barriers<br>created from the Devil Castle.<br><br>They are automatically summoned during the<br>Buster Phase and head towards the Devil Castle.<br><br><c ff4f3f>When all of the Buster Piles that are summoned<br>hit and destroy the Devil Barriers, or if they're destroyed,<br>the Buster Phase ends.<c>"""
```
Comment (Aethernelle) [inline Quests/Buster Quests/co_052103_01.csv]: I thought this line should be describing about Defense Phase.

Comment (Aida-Enna) [reply Quests/Buster Quests/co_052103_01.csv]: Ahhhh, this is what you meant. Good catch! I'll fix it, thanks.

File: UI/ui_item.csv [hunk 2]
```diff
@@ -634,7 +634,7 @@ UseItemBagMessage#0,"""This will unlock \""$(0)\""<br>and acquire the following
 UseItemBagError1#0,"""Insufficient space in inventory to receive items."""
 UseItemBagError2#0,"""Exceeded the max number of items that you can carry."""
 UseItemBagError3#0,"""Failed to acquire item."""
-ConsumeErrorMessage12#0,"""効果時間の上限を超えるため、<br>同じアイテムは使用できません。"""
+ConsumeErrorMessage12#0,"""You cannot use this item as<br>it would exceed the duration limit.""""
```
Comment (alama) [inline UI/ui_item.csv]: Why there an extra quote here?

Comment (Aida-Enna) [reply UI/ui_item.csv]: Because I was too tired, lol. I'll fix it, danke.

File: UI/ui_item.csv [hunk 3]
```diff
@@ -1741,15 +1741,15 @@ TitleReset#0,"""Select item to undo change"""
...
 NoItemMessageReset#0,"""外見を戻したい武器をセット"""
 TraceCheckTitle#0,"""Weapon Form Change Details"""
-TraceMessage#0,"""<yellow>武器フォーム変更パス：$(0)（現在：$(1)）<br>必要メセタ：$(2) （現在：$(3)）<c>"""
+TraceMessage#0,"""<yellow>Weapon Form Pass Needed: $(0)（Have: $(1)）<br>Meseta Needed: $(2) （Have: $(3)）<c>"""
 TraceCheckSubTitle1#0,"""Weapon Form"""
 TraceCheckSubTitle2#0,"""Required items"""
 TraceCheckSubTitle3#0,"""Weapon Form Change item"""
 ItemTraceCheckTitle#0,"""Confirm Weapon Form"""
-ItemTraceMessage#0,"""武器の外見を変更します。<br>よろしいですか？<br><br>※アイテムの性能は変わりません。<br>※ベースアイテムのフォトンカラーを<br>\u3000変更している場合、外見を元に戻しても<br>\u3000フォトンカラーがデフォルトの色に変更されます。<br>※外見を変更すると<br>\u3000ベースアイテムと外見に使用したアイテム両方が<br>\u3000トレードやマイショップの出品アイテムとして<br>\u3000選択できなくなります。"""
+ItemTraceMessage#0,"""This will change the appearance of the weapon.<br>Continue?<br><br>※The item stats will not change<br>※Photon color information<br>\u3000The photon color will bechanged to the default<br>\u3000even if it was changed.<br>※Once you change the appearance<br>\u3000Both the base item and the item changed<br>\u3000will be become untradeable."""
```
Comment (alama) [inline UI/ui_item.csv]: Now there a spelling "mistake", it is really "bechanged" or "be changed"?

Comment (Aida-Enna) [reply UI/ui_item.csv]: yee, already fixed that in the new push lol

**Comment by Aida-Enna** [comment]
y o l o

**Pull request merged**